styles/styles.scss

```diff
@@ -1999,4 +2001,15 @@ input[type="color"] {
 
 .modal-md {
   max-width: 500px;
+}
+
+/* Hide scrollbar for Chrome, Safari and Opera */
+.overflow-noscrollbar::-webkit-scrollbar {
```

Comment:
this can be joined to the next one,

```scss
.overflow-noscrollbar {
  &::-webkit-scrollbar { display: none; }
  -ms-overflow-style: none;
  scrollbark-width: none;
}
```